skins/laika/src/components/pages/frequently_asked_questions/Question.vue

```diff
@@ -1,11 +1,11 @@
 <template>
-	<div class="question" v-bind:class="[ isOpen ? 'accordion' : '' ]">
+	<div class="accordion-item" v-bind:class="[ isOpen ? 'accordion' : '' ]">
```

Comment:
Since I am re-using the same CSS classes, I have made them more neutral and moved them to another file.